### DISCUSSION TITLE
How to actually use the patterns configuration capability?

### DISCUSSION THREAD

**Discussion by handdara** [latex-lsp/texlab#939] [Q&A] [answered]
How do I actually insert the RegExp type into the config? is it just as a string? or is there some other way? Trying to get ignoring patterns working for the symbol finder.

**Comment by pfoerster** [comment, marked as answer]
@handdara It is just a string. Internally, it is converted into a regular expression using the [`regex`](https://docs.rs/regex/latest/regex/) crate. I will update the wiki page.
